simplebgc/src/commands/mod.rs

```diff
@@ -34,7 +34,7 @@ pub enum OutgoingCommand {
     BoardInfo3,
     Control(ControlData),
     MotorsOn,
-    MotorsOff { mode: Option<MotorsOffMode> },
+    MotorsOff { mode: Option<MotorsOffQuery> },
```

Comment:
Make this a tuple variant (`MotorsOff(MotorsOffQuery)`) for consistency

Reply:
done